__tests__/medium-zoom-test.js

```diff
@@ -126,10 +126,12 @@ describe('constructor', () => {
     root.appendChild(p)
 
     const options = {
-      margin: 24,
-      background: '#BADA55',
-      scrollOffset: 124,
-      metaClick: true
+      scrollOffset: 48,
+      metaClick: true,
+      overlayStyles: {
+        background: '#fff'
+      },
+      imgStyles: {}
```

Comment:
You should modify the original tests as little as possible.

So keep `scrollOffset: 124` and `margin: 24`